runtime/compute/cker/include/cker/Types.h

```diff
@@ -367,11 +367,11 @@ struct SliceParams
 struct StridedSliceParams
 {
   int8_t start_indices_count;
-  int16_t start_indices[4];
+  int32_t start_indices[5];
   int8_t stop_indices_count;
-  int16_t stop_indices[4];
+  int32_t stop_indices[5];
   int8_t strides_count;
-  int16_t strides[4];
+  int32_t strides[5];
```

Comment:
(Optional) The suggestion below is really trivial. You can ignore it.
I’d suggest grouping the three `int8_t` count fields together before the `int32_t` arrays to eliminate padding and shrink the struct from 72 to 64 bytes. That change requires updating any positional initializers for StridedSliceParams that make sure the brace‑list arguments match the new member order.
```suggestion
  int8_t stop_indices_count;
  int8_t strides_count;
  int32_t start_indices[5];
  int32_t stop_indices[5];
  int32_t strides[5];
```

Reply:
Okay, I'll check side effects and apply it. :)

Reply:
I've reviewed the code and I think it's better to keep the current code,
because I've found same structure definitions in several codes including onert-micro and even in luci.
The benefits with removing padding is not big, I think. So it's better to keep consistency.

Reply:
I see. Thanks for your review.

Reply:
OK, thanks for your review.